engine/main.py

```diff
@@ -640,6 +644,18 @@ def unsubscribe_group(group_name, user):
     logging.debug(res)
     return res
 
+def confirm_unsubscribe(membergroups, user):
+    res = True
+    admins = []
+
+    for m in membergroups:
+        if m.admin:
+            admins.append(m.member.email.encode("utf-8"))
+    if len(admins) == 1 and user.email in admins:
+        res = False
+    return res
```

Comment:
this function is nice. Just a minor improvement: naming of the function and a variable name _res_ might mislead. The function name could be something involving a word "check" or "can" rather than "confirm". Because this function is a predicate. Also, for res, something more informative